platform/core/src/services/HangingProtocolService/HangingProtocolService.js

```diff
@@ -213,9 +243,10 @@ class HangingProtocolService {
     // for each of the viewport matching procedures
 
     // Loop through each viewport
+    const alreadyMatched = [];
```

Comment:
@ladeirarodolfo This is the change that makes it possible to define hanging protocols that match subsequent images in a series.  The problem was that there wasn't a good way to say "match the SECOND display set" that matches.  So it is a fairly important fix to enable 2x2 layouts to actually work effectively.

Reply:
@sedghi  - can we get this file into your updated hanging protocol versions?  It just makes it easier to define HPs